### ISSUE TITLE
Warranty expire dates

### ISSUE THREAD

**Issue by sigbjornaib** [ManageIQ/manageiq-providers-lenovo#298]
LXCA looks up the warranty expire and displays this in the LXCA webui. Would it be possible to pull the warranty info from LXCA into ManageIQ?

**Comment by agrare** [comment]
That's interesting @sigbjornaib is that warranty per server?

**Comment by sigbjornaib** [comment]
Yes it is. You'll find it in LXCA under Adminstration -> Service and Support -> Warranty. 
Each server is listed with Start Date and Expiration Date of the warranty for that server. 

LXCA is using Lenovos SupportAPI service to look up the current warranty state for each server.

**Comment by agrare** [comment]
I don't have an LXCA system available but if you are able to pull that property back from the API I can help integrate that in.  I'll need to put in a schema migration to store the new property and add it to the collector and parser in this repo.
